src/token/impl/Cargo.toml

```diff
@@ -5,7 +5,7 @@ version = "0.2.0"
 
 [features]
 default = []
-no_api = []
+no_api = ["token-api/no_api"]
```

Comment:
There's a problem with this change. Take as an example the AMM canister. It depends on both `token-api` and `token-impl` crates. The first dependency is to use token API as is, to allow working with AMM supply as an IS20 token. And the second one is to make inter-canister calls to other IS20 canisters. This means that for AMM api crate should be used without `no_api` feature, and impl crate with it. But if we set the `no_api` on impl crate as suggested in this PR, there would be no way for AMM to turn off `no_api` feature for api create.